tripleblindmarket/private-kit
# fixed SAF-857 statusbar issue

#### Description:

Fixed the StatusBar backgroundColor and contentColor issue for android on ExplanationScreen, LocationPermissionScreen and Welcome screen

#### Linked issues:

https://pathcheck.atlassian.net/browse/SAF-857

#### Screenshots:
 ## Explanation Screen
![Screenshot_1596438364](https://user-images.githubusercontent.com/25247251/89155906-68933300-d587-11ea-9562-b75ffa15ebb8.png)

## LocationPermissionScreen
![Screenshot_1596438413](https://user-images.githubusercontent.com/25247251/89156139-d17aab00-d587-11ea-8d02-2367ed871488.png)


#### How to test:

1: Install the app / Clear cache and clear data from the app(Android only) 
2: Go through onboarding process 
3: Check the StatusBar is not in contrast color where page is blue and StatusBar color is white or vice versa
